Program.cs

```diff
@@ -1,4 +1,78 @@
-﻿using System;
-Console.WriteLine ("Hello, World!");
+﻿// --------------------------------------------------------------------------------------------
+// Training ~ A training program for new interns at Metamation, Batch - July 2023
+// Copyright (c) Metamation India.                                              
+// ------------------------------------------------------------------------
+// Program.cs
+// SORT AND SWAP SPECIAL CHARACTERS 
+// Given a character array A, along with special character S and sort order O (default order is Ascending)
+// Print the sorted array by keeping the elements matching S to the last of the array.
+// For example, Input=([a, b, c, a, c, b, d], a, “descending”) ;Output=d, c, c, b, b, a, a .
+// --------------------------------------------------------------------------------------------
+using System;
+using System.Linq;
+namespace Training {
+   #region Program ------------------------------------------------------------------------------
+   /// <summary>Sort and swap special character</summary>
+   internal class Program {
+      #region Methods ---------------------------------------------
+      /// <summary>Getting inputs from user</summary>
+      static void Main () {
+         // Getting character from the user.
+         char[] a = GetInput ("Enter the letters: ").ToCharArray ();
+         // Getting special character to swap from the user.
+         char s = char.Parse (GetInput ("Enter the special character: "));
+         // Getting the sorting order from the user. 
+         while (true) {
+            Console.Write ("Do you want to sort the character array in decending order: (y)es or (n)o: ");
+            ConsoleKeyInfo consoleKey = Console.ReadKey ();
+            ConsoleKey sortOrder = consoleKey.Key;

```

Comment:
You can just use `var key = Console.ReadKey ().Key` in a single line. No need to have a separate variable

Reply:
@Nandhini-Sakthi This change has not yet been done. Fix it. Always ensure you complete all the review suggestions before assigning for review.